app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt

```diff
@@ -265,7 +265,7 @@ class RegistrationFragment : BaseFragment() {
         )
     }
 
-    fun showError(msg: String?) {
+    fun showError(msg: Int) {
```

Comment:
you werent supposed to change the signature of these functions. You were supposed to handle them in the  ui in a way i described. 
We may remove the base activity and base fragment entirely in the future iterations and in such a case we would have to redo this everywhere.